CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## Unreleased
+* New debugging option: `l`. Adds a step after the current one.
```

Comment:
link to documentation please :) Of course, first needs an existing documentation for it :D